core/src/main/java/org/web3j/protocol/core/Request.java

```diff
@@ -75,4 +77,25 @@ public T send() throws IOException {
     public Observable<T> observable() {
         return new RemoteCall<>(this::send).observable();
     }
+
```

Comment:
Please can we use IntelliJ style equals & hashcode methods instead?

Reply:
Yeah, sure. Btw this one is generated by Intellij too, but requires java7(which is the problem for android, I guess)